src/templates/debug.html

```diff
@@ -7,8 +7,8 @@
 <section class="main full debug">
   <div>
     <button class="button" id="submit-debug">Submit logs</button>
-    <button class="button" id="enable-mkt-nav">
-      Enable new mkt-nav
+    <button class="button" id="flip-websites-waffle">
```

Comment:
No, I had started doing this before our meeting on Friday and then we decided to put it on altdev instead. I'll back it out.